aten/src/ATen/native/mkldnn/Matmul.cpp

```diff
@@ -53,17 +78,29 @@ static bool use_mkldnn_bf16_matmul() {
       mkldnn_bf16_device_check());
 }
 
-bool mkldnn_bf16_gemm(
+static bool use_mkldnn_fp16_matmul() {
+  return (
+      at::globalContext().userEnabledMkldnn() &&
+      mkldnn_fp16_device_check());
```

Comment:
Please remove extraneous brackets (not sure about the rest of the formatting)
```suggestion
  return at::globalContext().userEnabledMkldnn() &&
      mkldnn_fp16_device_check();
```

Reply:
Revised as suggested